lib/sparrow/pools_warden.ex

```diff
@@ -81,11 +96,17 @@ defmodule Sparrow.PoolsWarden do
         "worker=pools_warden, action=init, result=success"
       end)
 
+    :telemetry.execute(
+      [:sparrow, :pools_warden, :init],
+      %{args: %{}},
```

Comment:
Why do we put those empty values here? Also the state below is redundant.

Reply:
Forgot about `PoolsWarden`'s `telemetry` calls during refactoring